README.md

```diff
@@ -4,16 +4,42 @@ Language server for Ralph.
 
 Currently supports text document level events and diagnostics.
 
+# Installation
```

Comment:
WDYT of moving this after `Install VSCode plugin` and or within `Install Neovim plugin`? Most devs probably just want to download plugins for their IDEs and this installation step is not required for VSCode. But it is required for VIM.